### PR TITLE
add feed aggregator proposal

### PR DESCRIPTION
**Pull request by mikeosborne**

### CODE REVIEW

File: content/grants/proposals/feed-aggregator.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+reward = 5
+assignee = "~mirtyl-wacdec"
+grant_id = "P0043"
+completed = false
+link = ""
++++
+
+### Project Name
+
+Urbit Feed Aggregator
```
Comment (jalehman) [inline content/grants/proposals/feed-aggregator.md]: This isn't necessary. Let's remove it.

File: content/grants/proposals/feed-aggregator.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+## Purpose
+
+Urbit as it stands now (June 2021) is a very functional social network, but it's not yet making use of all the strengths of its computing model.
+As a personal server Urbit is the perfect platform to build a centralized social media consumption application.
```
Comment (jalehman) [inline content/grants/proposals/feed-aggregator.md:20]: Assuming they mean "decentralized" here?

```suggestion
As a personal server Urbit is the perfect platform to build a decentralized social media consumption application.
```

File: content/grants/proposals/feed-aggregator.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+Most importantly, though UTw can fetch data from external sources and present them for consumption, and interaction with the Microblogging app.
+
+Most internet users subscribe to a variety of data sources. Twitter, Facebook, Reddit, YouTube; this or that internet forum. Maybe some RSS feeds.
+You will be able to import that data (scraping if public, fetching it if authenticated) and keep it as a column in the interface. You will also be able to share that data on your microblogging feed; "retweeting" or quoting data from any source you're subscribed to.
+
+The interface (as of now) follows TweetDeck; a deck of multiple columns, each displaying data from a single source, with vertical scroll.
+
+"Alternative frontends" such as Nitter or Invidious exist for many mainstream applications, but they all have one big problem: rate-limiting. Urbit solves this issue by having individual users do their own data fetching. No application can possibly block individual users fetching data for normal, personal use. That is literally a problem that only Urbit is equipped to solve.
+
+* Target User
```
Comment (jalehman) [inline content/grants/proposals/feed-aggregator.md]:
```suggestion
### Target User
```

File: content/grants/proposals/feed-aggregator.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+Target user is any social media user who wants their data in a single dashboard, most especially people who like microblogging and would enjoy a way of easily sharing or commenting on their own data to their followers.
+
+* What does it enable to be further built afterwards, and how does this advance Urbit's strategic goals?
```
Comment (jalehman) [inline content/grants/proposals/feed-aggregator.md]:
```suggestion
### What does it enable to be further built afterwards, and how does this advance Urbit's strategic goals?
```

File: content/grants/proposals/feed-aggregator.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+A plugin system will allow data fetching and interaction from virtually infinite internet data sources. This achieves one of the earliest goals of Urbit, aggregation of internet data in a single private interface, where data is under your own control. It also essentially kills Twitter; it replicates all its functionality, and then some.
+
+## End Product
+
+* Urbit code produced
```
Comment (jalehman) [inline content/grants/proposals/feed-aggregator.md]:
```suggestion
### Urbit code produced
```

File: content/grants/proposals/feed-aggregator.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+UTw would be an external application, to be installed as a third-party package once the capability exists.
+It does not require to be merged into core but it does require to extend the present functionality of graph-store, e.g. a timeline generator (scrying multiple resources at once) would be required, and that capability would be useful, we believe, to merge into core.
+
+* Non-Urbit binaries or code produced
```
Comment (jalehman) [inline content/grants/proposals/feed-aggregator.md]:
```suggestion
### Non-Urbit binaries or code produced
```

File: content/grants/proposals/feed-aggregator.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+Beta version will be released as an external Elixir web-app that connects to Urbit via Airlock. Final version should be a lansdcape-ish React frontend on a (minimal) hoon backend
+
+* Documentation
```
Comment (jalehman) [inline content/grants/proposals/feed-aggregator.md]:
```suggestion
### Documentation
```

**Review by jalehman — CHANGES_REQUESTED** [review]
Some formatting nits.

File: content/grants/proposals/feed-aggregator.md (hunk 2 above)
Comment (mikeosborne) [reply content/grants/proposals/feed-aggregator.md:20]: I think centralized is correct. The data from the disparate feeds is centralized on your Urbit server.

**Review by jalehman — APPROVED** [review]

**Pull request merged**